indico/modules/events/registration/templates/management/regform_list.html

```diff
@@ -12,6 +12,29 @@
 {% endblock %}
 
 {% block content %}
+    {% if not event.has_feature('payment') %}
+        <div id="payment-disabled-notice" class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-coins"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Payments disabled{% endtrans %}
+                    </div>
+                    {% trans %}Payments are disabled for this event{% endtrans %}
+                </div>
+                <div class="toolbar right">
+                    <a href="#" class="js-enable-payments"
+                       data-href="{{ url_for('event_features.switch', event, feature='payment') }}"
```

Comment:
Although switch is the element in features, toggle is the action. Who should I blame for lousy naming? :P